_install-and-configure/install-opensearch/rpm.md

```diff
@@ -27,7 +27,7 @@ Generally speaking, installing OpenSearch from the RPM distribution can be broke
 1. **Configure OpenSearch for your environment.**
    -  Apply basic settings to OpenSearch and start using it in your environment.
 
-The RPM distribution provides everything you need to run OpenSearch inside Red Hat or Red Hat–based Linux Distributions. For a list of supported operating systems, see [Operating system compatibility]({{site.url}}{{site.baseurl}}/install-and-configure/install-opensearch/index/#operating-system-compatibility).
+The RPM distribution provides everything you need to run OpenSearch inside Red Hat or Red Hat–based Linux Distributions. For a list of supported operating systems, see [Operating system compatibility]({{site.url}}{{site.baseurl}}/install-and-configure/os-comp/).
```

Comment:
Should we add a similar one for the OSD part as well?
Just wondering.

Reply:
Since we moved the new page applies to both systems, I don't think its necessary.